frontend/src/components/relation-map/relation-map.css

```
@@ -0,0 +1,375 @@
+  width: 24px;
+  height: 2px;
+}
+
+.legend-line.solid {
+  height: 2px;
+}
+
+.legend-line.dashed {
+  background: repeating-linear-gradient(
+    90deg,
+    currentColor 0,
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)